.github/workflows/release_branches.yml

```diff
@@ -6,7 +6,10 @@ env:
   AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
   AWS_DEFAULT_REGION: ${{ secrets.AWS_DEFAULT_REGION }}
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-  REGRESSION_RESULTS_URL: altinity-build-artifacts/${{github.event.number}}/$GITHUB_SHA
+  REGRESSION_RESULTS_URL: ${{github.event.number}}/${GITHUB_SHA}/testflows
```

Comment:
Where would that end up on s3 bucket?

Reply:
It would go to altinity-build-artifacts/${{github.event.number}}/${GITHUB_SHA}/testflows.
Currently the testflows logs are uploaded to `altinity-test-reports`, but `artifacts=builds` will change the upload script to user the altinity-build-artifacts bucket instead.